init.lua

```diff
@@ -14,7 +14,7 @@ modular_computers.mod = {}
 modular_computers.mod.name = minetest.get_current_modname()
 modular_computers.mod.path = minetest.get_modpath(modular_computers.mod.name)
 
-modular_computers.registered_commands = {}
+modular_computers.os.registered_commands = {}
```

Comment:
IMO this can be moved into `src/os/api/cli.lua`. Then it is possible to create `register_command` and `registered_commands` together, such as:
```lua
modular_computers.os.register_command, module_computers.os.registered_commands = make_registration()
```
with
```lua
function make_registration()
  local t = {}
  return t, function(k, v) t[k] = v end
end
```

Then such a "registration creator" can be reused for other purposes in the future.

Reply:
That's a little abstract and I don't really see where that could be used, so I don't see any purpose in that.

Reply:
The suggestions mainly comes from my assumption that it will be possible to (globally) register other things than commands. I personally do not consider the abstraction a problem in itself.

That said, I agree that this is currently not needed.